about.html

```diff
@@ -34,114 +34,25 @@ <h2>Project Leads</h2>
 
-    <li class="asheesh">
-      <h3>Asheesh Laroia</h3>
-      <h4>Software Engineer (emeritus)</h4>
-      <p>During college, Asheesh organized the Xbox Linux project’s documentation into a wiki, submitted written testimony for the Electronic Frontier Foundation’s OPG v. Diebold case, revived the Johns Hopkins ACM chapter, and served as web team lead for Students for Free Culture. After graduation, he engineered software Creative Commons and the Participatory Culture Foundation and co-founded OpenHatch, a non-profit that helps people get involved in open source software. In 2011, he co-founded the Boston Python Workshop for women and their friends and delivered a talk at PyCon that triggered the formation of PyLadies. In 2013, he took a one-year vacation from free software-related work to hone his skills as a Security Engineer at Eventbrite. Asheesh is a Developer in Debian and holds a B.A. in cognitive science, with minors in women &amp; gender studies and French literature, and an M.S.E. in computer science.
-      <h5>Co-founder, <a href="https://openhatch.org/">OpenHatch</a><br>Security Engineer, <a href="https://www.eventbrite.com/">Eventbrite 2013-2015</a><br>Now working at <a href="https://stripe.com">Stripe</a></h5>
+<p>In early 2017, Sandstorm Development Group <a href="https://sandstorm.io/news/2017-02-06-sandstorm-returning-to-community-roots">ran out of funding</a> and the team primarily <a href="https://sandstorm.io/news/2017-03-13-joining-cloudflare">joined Cloudflare</a>. Kenton built off of Cap'n Proto, which underpins much of Sandstorm's inner workings, and created Cloudflare Workers. Sandstorm development continued with a major rewrite of the HTTP proxy, an identify refactor, and the addition of localization support. However, the pace of development slowed, and the Oasis <a href="https://sandstorm.io/news/2018-08-27-discontinuing-free-plan">free plan was discontinued</a> in 2018. Finally, <a href="https://sandstorm.io/news/2019-09-15-shutting-down-oasis">Oasis shut down</a> for paying customers at the end of 2019.</p>

```

Comment:
"an identify refactor" feels sort of "insider" terminology to me (or maybe this is a generic term I should know about?). Maybe it could be made more clear for the outside world. Could this be something like "a refactor of the identify system", or maybe "a refactor of the Identify system"?

Reply:
Well it was "identity" so that's a typo for one, but yeah, this is a technical change that probably doesn't warrant significant inclusion on the about page.